include/ars408_ros/ars408_commands.hpp

```diff
@@ -0,0 +1,673 @@
+// Copyright 2021 Perception Engine, Inc. All rights reserved.
```

Comment:
Did you use `git mv`? :thinking: 
It seems not to be recognized as the same file.
![image](https://user-images.githubusercontent.com/31987104/126054619-84eb995e-0ba6-4d1a-9d03-30deb903eff6.png)

Reply:
Hmm, perhaps we should split PRs to show the diff correctly... :pleading_face:

Reply:
I'm okay if you want to merge as it is.

Reply:
I split PR. https://github.com/tier4/ars408_driver/pull/3

Reply:
God.